tests/emulators/default_q0/platform.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import pathlib
+
+from qibolab.channels import ChannelMap
+from qibolab.instruments.emulator.models import general_no_coupler_model
+from qibolab.instruments.emulator.models.methods import load_model_params
+from qibolab.instruments.emulator.pulse_simulator import PulseSimulator
+from qibolab.platform import Platform
+from qibolab.serialize import load_qubits, load_runcard, load_settings
+
+log = logging.getLogger()
+log.setLevel(logging.INFO)  # log.setLevel(logging.ERROR)
+
+FOLDER = pathlib.Path(__file__).parent
+
+simulation_config = {
+    "simulation_backend_name": "Qutip",
+    "default_nshots": 100,
+    "sim_sampling_boost": 10,  # 20
+    "simulate_dissipation": True,
+    "instant_measurement": True,
+}
+
+
+def create(nlevel: int = 3):
```

Comment:
To comply with the `create_platform` interface, platform `create` methods should not have arguments. This could be moved to `parameters.json` as an "instrument setting".

Reply:
Fixed in 4e6cc09b9198fb0e218e0dd08a5f539dad122e19 and 7f4b9230dd3d7291758bb3681abb75570ec34bf6